web/src/tippyjs.js

```diff
@@ -185,13 +185,33 @@ export function initialize() {
         e.currentTarget._tippy.destroy();
     });
 
+    delegate("body", {
+        target: ".slow_send_spinner",
+        appendTo: () => document.body,
+        onShow(instance) {
+            instance.setContent(
+                $t({
+                    defaultMessage:
+                        "Your message is taking longer than expected to be sent. Sending...",
+                }),
+            );
+        },
```

Comment:
I wasn't able to get the tooltip to stay visible after the message has been sent and re-rendered but I think it is possible to have this tooltip be still visible. @timabbott should we add a mutation observer here or just hide the tooltip if it exists when calling `reify_message_id`?

Reply:
@lakshaykananiya let's add a `MutationObserver` here to ensure that the tooltip does hide. You can look at `do_render_buddy_list_tooltip` as an example of how to use it in this case.

Reply:
This is the only major change remaining in the PR.

Reply:
@lakshaykananiya I added it already.

Reply:
By using `hide_tooltip_if_reference_removed` function of tippy.js.

Reply:
Great!